db/migrations/0001_squashed_0004_alter_order_options.py

```diff
@@ -0,0 +1,141 @@
+# Generated by Django 4.0.2 on 2024-07-24 20:56
```

Comment:
There is no need to have all those migration files. Remove them and create only one which will represent all changes to db